### PR TITLE
[FEC-49] test-data-migration-batch12-models 

### PR DESCRIPTION
**Pull request by antoniolodias**
#### Summary

This PR creates the necessary models for this [migration PR](https://github.com/commercetools/merchant-center-frontend/pull/17434) which is based on [batch 12](https://commercetools.atlassian.net/browse/FEC-49) of the [chapter epic](https://commercetools.atlassian.net/browse/FEC-26) where we are migrate tests to use public test-data package 

#### Description

This PR unfortunately got a bit bigger than expected, it tries to create and fix the models needed on the migration PR

NOTE: still a WIP.

### CODE REVIEW

File: models/cart/src/shipping-info/transformers.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  default: Transformer<TShippingInfo, TShippingInfo>('default', {
+    buildFields: [
+      'taxCategory',
+      'shippingMethod',
+      'price',
+      'shippingRate',
+      'taxedPrice',
+      'taxRate',
+      'deliveries',
+      'discountedPrice',
```
Comment (jaikumar-tj) [inline models/cart/src/shipping-info/transformers.ts:14]: nit: we can move this to a constant within this file and reuse it

File: models/order/src/parcel/generator.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import type { Parcel } from '@commercetools/platform-sdk';
+import { fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { DeliveryItem } from '../delivery-item';
+import { ParcelMeasurements } from '../parcel-measurements';
+import { TrackingData } from '../tracking-data';
+
+const [getOlderDate] = createRelatedDates();
+
+// https://docs.commercetools.com/api/projects/orders#Parcel
```
Comment (jaikumar-tj) [inline models/order/src/parcel/generator.ts:10]:
```suggestion
// https://docs.commercetools.com/api/projects/orders#parcel
```
The capitalized `P` doesn't navigate to the parcel section

File: models/product-type/package.json [hunk 3]
```diff
@@ -8,20 +8,30 @@
...
+    "LICENSE",
+    "README.md"
+  ],
   "dependencies": {
     "@babel/runtime": "^7.17.9",
     "@babel/runtime-corejs3": "^7.17.9",
     "@commercetools-test-data/commons": "10.2.0",
     "@commercetools-test-data/core": "10.2.0",
     "@commercetools-test-data/utils": "10.2.0",
+    "@commercetools-test-data/type": "10.1.1",
```
Comment (jaikumar-tj) [inline models/product-type/package.json:34]: Just a reminder to update the version of this package, in-sync with other packages before merging this PR. The latest version is `10.3.0`

**Review by jaikumar-tj — COMMENTED** [review]
💯 Thanks for not losing your mind while migrating all these files 😅

**Comment by antoniolodias** [comment]
this PR will be closed and divided between
https://github.com/commercetools/test-data/pull/664
https://github.com/commercetools/test-data/pull/666
https://github.com/commercetools/test-data/pull/667
https://github.com/commercetools/test-data/pull/668
https://github.com/commercetools/test-data/pull/669

**Pull request closed without merging**